src/components/ChallengePage/ChallengePage.jsx

```diff
@@ -0,0 +1,65 @@
+import React, { useState } from "react";
+import { mockChallenges } from "../../mock-data/mockChallenges";
+import ChallengeCard from "../ChallengeCard/ChallengeCard";
+import DashboardNavbar from "../../components/DashboardNavbar/DashboardNavbar";
+import filterIcon from "../../assets/images/filterIcon.svg";
+import "./ChallengePage.scss";
+
+const ChallengePage = () => {
+  const [selectedCategories, setSelectedCategories] = useState([
```

Comment:
I think you just need an array, not a state. I don't see you changing this array anywhere